server/models/supplierAssessmentAppointment.ts

```diff
@@ -0,0 +1,12 @@
+import AppointmentAttendance from './appointmentAttendance'
+import AppointmentBehaviour from './appointmentBehaviour'
+
+export default interface SupplierAssessmentAppointment {
```

Comment:
Would there be any value in having a shared interface e.g. `Appointment` that the two types of appointment inherit/extend from and, in future, add specific fields to?

Reply:
Yeah, that's a good idea. I think I'll actually need that so I can have a single presenter that works for both the pages, in my next PR.

Reply:
I've got rid of the `SupplierAssessmentAppointment` and replaced it with `Appointment`.